ui/js/models/resultsets_store.js

```diff
@@ -1182,56 +1178,24 @@ treeherder.factory('ThResultSetStore', [
                                 return matcher;
                             }
                         );
-                    }
-                );
-                var symbolMap = _.object(paddedJobSymbolList, jobSymbolList);
-                paddedJobSymbolList.sort();
-                var outputOrderMap = {};
-                _.each(paddedJobSymbolList, function(paddedJob, index){
-                    outputOrderMap[symbolMap[paddedJob]] = index;
+                    });
+                });
+                platform.groups = _.sortBy(platform.groups, function(group) {
+                    if (!group.symbol.length)
+                        return undefined;
```

Comment:
usually preferred to have {} braces around this if-block, even though it's one line.